packages/generators/init-generator.ts

```diff
@@ -142,11 +142,18 @@ export default class InitGenerator extends Generator {
 		}
 
 		const { langType } = await this.prompt([
+			// At this point of time we don't know that it will be adding a plugin
```

Comment:
We don't, but we can write code on client to handle this. Other than that, I think that making a scaffold starter in specific for UI would do good

Reply:
Okay makes sense, than what about starters available at npm ? Their's generator can't be edited by us for that info.

Reply:
No, but if they fill inn say for instance `typescript`, then the author would add the loaders etc needed in the background and since we're reflecting the webpack config object about to be generated at UI, this is fine

Reply:
Okay, then maybe we can chill this feature untill we have final designs and the migration of features is completed.